pkg/cmd/component/cmd/ui/ui.go

```diff
@@ -183,6 +183,7 @@ func AddTargetPort() int {
 	var targetPort int
 	prompt := &survey.Input{
 		Message: "Provide the Target Port ",
+		Help:    "",
```

Comment:
Provide the similar text that you made to the description of the flag from above

Reply:
Okay !